shared/components/empty-content/empty-content.jsx

```diff
@@ -5,6 +5,11 @@ var React = require( 'react' ),
 	debug = require( 'debug' )( 'calypso:components:emptyContent' ),
 	classNames = require( 'classnames' );
 
+/**
+ * Internal dependencies
+ */
+var FeatureExample = require( 'components/feature-example' );
```

Comment:
I'd much rather we do this compositionally from the page that needs it instead. This seems a bit convoluted.